wordpress-mobile/WordPress-iOS
# Feature Introduction: configure action buttons

Ref: #18176

Configure the action buttons for Feature Introduction.

Because we don't have any official way of showing the feature introduction just yet, I have set it to show in `applicationDidBecomeActive` to facilitate easily accessing it (by minimizing and relaunching the app). This is temporary and will be removed before merging. 

To test:

---
Feature Introduction buttons:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction should appear. 
- Verify:
  - Two buttons are shown at the bottom.
  - They are stacked vertically.
  - The primary (blue) button is on top.

<kbd>![fi_buttons](https://user-images.githubusercontent.com/1816888/160907828-cb9d7d11-0fe6-4cb2-8daa-c941fab5088c.png)</kbd>

---
Other buttons:
- Verify there are no regressions in the button display in other uses of `CollapsableHeaderViewController`. A good example is Site Page creation (it's also used in WP site creation).
- Tap on the FAB and select `Site page`.
- Verify the `Create Blank Page` button appears.
- Select a layout.
- Verify the `Preview` and `Create Page` buttons appear horizontally.

| ![layout_default](https://user-images.githubusercontent.com/1816888/160908440-664001a3-5b61-4726-b1b5-b1d3f2aae70c.png) | ![layout_selected](https://user-images.githubusercontent.com/1816888/160908436-0893578b-3415-41e9-883b-e286452de70f.png) |
|--------|-------|

---
## Regression Notes
1. Potential unintended areas of impact
Button display in other uses of `CollapsableHeaderViewController`.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Compared button display in other uses of `CollapsableHeaderViewController`. Looks the same to me!

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
